client/src/extension.ts

```diff
@@ -392,6 +390,34 @@ Executable ${this.denoInfo.executablePath}`;
       this.client.sendNotification(Notification.diagnostic, uri.toString());
     }
   }
+
+  private isPartOfProject(
+    config: ConfigurationField,
+    entry_path: string,
+    workspace_path?: string
+  ): boolean {
+    if (workspace_path === undefined) {
+      return false;
+    }
+
+    const includes =
```

Comment:
@lucacasonato you can add a breakpoint at this line to see what's going wrong, I can't reproduce what you've encountered